scripts/boot/zub1cg/boot_qspi_mmc.sh

```diff
@@ -15,8 +15,6 @@
 #  This design is the property of Avnet.  Publication of this
 #  design is not authorized without written consent from Avnet.
 #
-#  Please direct any questions to the PicoZed community support forum:
-#     http://avnet.me/zub1cg_forum
```

Comment:
We want to keep these lines in this file.  I will fix the avnet.me/zub1cg_forum link.

Reply:
I'll update this one and update the other ones in this and the other repos when I get more stuff merged back in.